pkgs/servers/web-apps/lemmy/ui.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, mkYarnPackage
+, libsass
+, nodejs
+, python3
+, pkg-config
+, writeShellScriptBin
+, jq
+, runCommand
+}:
+
+let
+  pkgConfig = {
+    node-sass = {
+      nativeBuildInputs = [ ];
+      buildInputs = [ libsass pkg-config python3 ];
+      postInstall = ''
+        LIBSASS_EXT=auto yarn --offline run build
+        rm build/config.gypi
+      '';
+
+      dontFixup = true;
+
+    };
+  };
+
+  pname = "lemmy-ui";
+  version = "0.12.2";
+  src = builtins.fetchGit {
+    url = "https://github.com/LemmyNet/lemmy-ui";
+    rev = "cb1df009c10f089753b47c0e2398acf8d594ba54";
+    ref = "main";
+    submodules = true;
+  };
```

Comment:
Maybe we can just use `fetchFromGitHub`, check how other packages are doing this (for example we can use `rev = version;` and `fetchSubmodules = true;`).

Reply:
Thank you for this!